### ISSUE TITLE
FMT_SMF_EXT.2.1 remediation actions

### ISSUE THREAD

**Issue by woodbe** [commoncriteria/mobile-device#34]
https://github.com/commoncriteria/mobile-device/blob/954ded4330200e30c1e4b35b7b7398b6d6a3598c/input/mobile-device.xml#L6780

The "are" should be "may".

The actions list here are available to the MDM Agent, but that does not mean that the MDM Agent actually calls any of them. Some actions may be automatic, like unenrolling would remove Enterprise data, but other things may be available, but are not specifically called by an MDM Agent.

Basically saying these actions "are" makes it sound like anything listed here WILL be done, whereas these are all things the MDM Agent could do, so "may" is more accurate in how this actually works.

**Comment by lewyble** [comment]
Done.
